### PR TITLE
Lite build support

### PR DESCRIPTION
**Pull request by lllwan**

### CODE REVIEW

File: build/lite/charts/charts.go [hunk 1]
```diff
@@ -34,8 +48,6 @@ func (charts *Charts) ListImages(clusterName string) ([]string, error) {
 			list = append(list, images...)
 		}
 	}
-
-	list = removeDuplicate(list)
```
Comment (fanux) [inline build/lite/charts/charts.go:38]: Why delete this?

Comment (lllwan) [reply build/lite/charts/charts.go:38]: Image pull will be de duplicated. Here, de duplication is equivalent to two operations. The actual pull is to parse the merge data of the helm chart and read the image. The de duplication here only deals with the data of helm chart

File: sealer/cmd/build.go [hunk 2]
```diff
@@ -65,6 +68,7 @@ func init() {
 	buildCmd.Flags().StringVarP(&buildConfig.BuildType, "buildType", "b", "", "cluster image build type,default is cloud")
 	buildCmd.Flags().StringVarP(&buildConfig.KubefileName, "kubefile", "f", "Kubefile", "kubefile filepath")
 	buildCmd.Flags().StringVarP(&buildConfig.ImageName, "imageName", "t", "", "cluster image name")
+	buildCmd.Flags().StringVarP(&buildConfig.ImageName, "imageList", "i", "", "build by submitting image list")
```
Comment (fanux) [inline sealer/cmd/build.go:71]: Just using $rootfs/manifests/imageList file, for most users don't care about this argument.

**Comment by fanux** [comment]
How do you save the new container images to the CloudImage layer?

File: sealer/cmd/build.go (hunk 2 above)
Comment (lllwan) [reply sealer/cmd/build.go]: OK

**Review by fanux — APPROVED** [review]

**Pull request merged**